includes/products/class-newspack-listings-products-cron.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Newspack Listings - Handles automated cron jobs and other scheduled tasks.
+ * Mainly for expiring single-purchase listings.
+ *
+ * @package Newspack_Listings
+ */
+
+namespace Newspack_Listings;
+
+use \Newspack_Listings\Newspack_Listings_Settings as Settings;
+use \Newspack_Listings\Newspack_Listings_Core as Core;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Products class.
+ * Sets up WooCommerce products for listings.
+ */
+final class Newspack_Listings_Products_Cron extends Newspack_Listings_Products {
```

Comment:
By using the `Newspack_Listings` namespace, you can take advantage of it for cleaner class and file names:

The file names would also change from `class-newspack-listings-products-cron.php` to `class-products-cron.php`.

```suggestion
namespace Newspack_Listings;

use \Newspack_Listings\Settings;
use \Newspack_Listings\Core;

defined( 'ABSPATH' ) || exit;

/**
 * Products class.
 * Sets up WooCommerce products for listings.
 */
final class Products_Cron extends Products {
```

Reply:
Great idea! This is much cleaner and I'm not sure why I didn't think of it sooner. Implemented in d618b29.

Reply:
Awesome! I love how clean it looks.

There's just one more mention of `\Newspack_Listings\Newspack_Listings_Settings` at line `132` of `class-core.php`

Reply:
Good catch! Updated in 0ad3afb. This also prompted me to do a sweep in other repos for Listings dependencies, and I found one instance in Blocks which will need to be updated concurrently with this refactor. PR here: https://github.com/Automattic/newspack-blocks/pull/1064